### PR TITLE
[command] update evaluate proto

### PR DESCRIPTION
**Pull request by fenrir-z**

### CODE REVIEW

File: ymir/command/proto/mir_command.proto [hunk 1]
```diff
@@ -216,6 +216,7 @@ message SingleDatasetEvaluation {
 message SingleIouEvaluation {
```
Comment (phoenix-xhuang) [inline ymir/command/proto/mir_command.proto:216]: average_evaluation也要改名

File: ymir/command/proto/mir_command.proto [hunk 2]
```diff
@@ -216,6 +216,7 @@ message SingleDatasetEvaluation {
...
     map<int32, SingleTopicEvaluation> ci_evaluations = 1;  // key: class ids
     map<string, SingleTopicEvaluation> topic_evaluations = 2;  // key: topic names
+    SingleTopicEvaluation average_ci_evaluation = 3;  // evaluations averaged by class ids
```
Comment (phoenix-xhuang) [inline ymir/command/proto/mir_command.proto]: average_ci_evaluation放到2

Comment (IJtLJZ8Rm4Yr) [inline ymir/command/proto/mir_command.proto]: 上面的 `average_evaluation` 是不是要 rename

Comment (fenrir-z) [reply ymir/command/proto/mir_command.proto]: done

Comment (fenrir-z) [reply ymir/command/proto/mir_command.proto]: done

File: ymir/command/proto/mir_command.proto (hunk 1 above)
Comment (fenrir-z) [reply ymir/command/proto/mir_command.proto:216]: done

**Review by phoenix-xhuang — APPROVED** [review]

**Review by IJtLJZ8Rm4Yr — APPROVED** [review]

**Pull request merged**